app/models/concerns/recordable.rb

```diff
@@ -13,13 +13,15 @@ module Recordable
   # Centralises the logic around deciding what points to score for an item
   #
   # Including classes should implement: a +score+ method or attribute
-  def score_when_recorded_at(school, date)
-    academic_year = school.academic_year_for(date)
+  def score_when_recorded_at(observation)
```

Comment:
wonder about the method name, maybe calculate_score would be clearer?

Reply:
Yes, that would be better (certainly now the params have changed), thanks